CRM/Import/DataSource/CSV.php

```diff
@@ -270,3 +273,14 @@ function civicrm_mysql_real_escape_string($string) {
   }
   return $dao->escape($string);
 }
+
+/**
+ * @todo Find a better place for this function. Maybe utils?
+ * @param $string
+ *
+ * @return string
+ *    Return the nbsp; trimmed version of the string.
+ */
+function trim_nbsp($string) {
+  return trim($string, chr(0xC2) . chr(0xA0));
+}
```

Comment:
Instead of placing this function in the global scope, I would suggest putting it in this class (or CRM_Utils_String). Or maybe easiest & simplest for now would be to make it an anonymous function and pass it straight into array_map.